versionsupport_v1_20_r3/src/main/java/com/tomkeuper/bedwars/support/version/v1_20_R3/v1_20_R3.java

```diff
@@ -369,12 +372,12 @@ public void registerTntWhitelist(float endStoneBlast, float glassBlast) {
             field.set(Blocks.aQ, glassBlast);
 
             var coloredGlass = new net.minecraft.world.level.block.Block[]{
-                    Blocks.ej, Blocks.ek, Blocks.el, Blocks.em,
-                    Blocks.en, Blocks.eo, Blocks.ep, Blocks.eq,
-                    Blocks.er, Blocks.es, Blocks.et, Blocks.eu,
-                    Blocks.ev, Blocks.ew, Blocks.ex, Blocks.ey,
+                    Blocks.ei, Blocks.ej, Blocks.ek, Blocks.el,
+                    Blocks.em, Blocks.en, Blocks.eo, Blocks.ep,
+                    Blocks.eq, Blocks.er, Blocks.es, Blocks.et,
+                    Blocks.eu, Blocks.ev, Blocks.ew, Blocks.ex,
```

Comment:
Why did you change these?

Reply:
Those are the glass objects, previous ones were wrong.

Reply:
Ahh I see, good catch!